src/main/scala/QuartzSchedulerTypedExtension.scala

```diff
@@ -0,0 +1,97 @@
+package com.typesafe.akka.extension.quartz
+
+import java.util.{Date, TimeZone}
+
+import akka.actor.typed.{ActorRef, ActorSystem, ExtensionId}
+
+object QuartzSchedulerTypedExtension extends ExtensionId[QuartzSchedulerExtension] {
```

Comment:
I don't think you get much by extending `ExtensionId[QuartzSchedulerExtension]`. Wouldn't this be better as `ExtensionId[QuartzSchedulerTypedExtension]`? You'd need to override `lookup` and `get` too.

Reply:
@enragedginger I've implemented the `get` method, but `lookup` doesn't exist in `Akka Typed`.
Can you check this info? Even the documentation is implementing without `lookup` method.

![Captura de Tela 2020-11-18 às 12 39 36](https://user-images.githubusercontent.com/1410319/99551864-20174e00-299b-11eb-8578-b58e68d970ba.png)